OrbitQt/orbitmainwindow.cpp

```diff
@@ -679,24 +679,31 @@ void OrbitMainWindow::on_actionSave_Capture_triggered() {
 //-----------------------------------------------------------------------------
 void OrbitMainWindow::on_actionOpen_Capture_triggered() {
   QString file = QFileDialog::getOpenFileName(
-      this, "Open capture...", Path::GetCapturePath().c_str(), "*.orbit");
+      this, "Open capture...", QString::fromStdString(Path::GetCapturePath()),
+      "*.orbit");
   if (file.isEmpty()) {
     return;
   }
 
+  (void) OpenCapture(file.toStdString());
```

Comment:
what happens when an error occurs and the result is discarded like that? Does it raise an exception?

Reply:
No exception is raised. The (void) is just needed to make the nodiscard attribute happy.